docs/references/ayx-plugin-cli.md

```diff
@@ -215,14 +215,25 @@ The plugin version.
 -   Type: String
 -   Permissible Values: Any
 
+#### \--omit-ui
+
+Do not generate UI artifacts for this plugin. UI artifacts can be generated later by using `generate-ui` command.
+
+
 ## create-yxi
 
 This command packages a workspace into a YXI that can be installed into
 Designer. The resulting YXI is located under `build/yxi/`.
 
 ### Parameters
 
-This command takes no parameters.
+#### \--omit-ui
+
+Do not build any UI artifacts for the plugins in the workspace.
+
+- If a plugin does not have any UI artifacts, nothing will happen.
+- If a plugin has UI artifacts but it was never built, nothing will happen.
+- If a plugin has UI artifacts that was previously built, the previous build result will be packaged.
```

Comment:
TW review:

Don't build any UI artifacts for the plugins in the workspace.

- If a plugin doesn't have any UI artifacts, nothing happens.
- If a plugin has UI artifacts, but it was never built, nothing happens.
- If a plugin has UI artifacts and was previously built, the previous build result is packaged.